apps/src/lib/levelbuilder/rubrics/RubricEditor.jsx

```diff
@@ -8,6 +8,7 @@ export default function RubricEditor({
   deleteLearningGoal,
   learningGoalList,
   updateLearningGoal,
+  disabled,
```

Comment:
Is this needed? In the screenshot it looks like this component isn't loaded when there aren't submittable levels?

Reply:
Good point - this does mean this is no longer needed.  I'll make the change